sdk/keyvault/Azure.Security.KeyVault.Keys/src/KeyType.cs

```diff
@@ -53,6 +54,11 @@ public KeyType(string value)
         /// </summary>
         public static KeyType Oct { get; } = new KeyType(OctValue);
 
+        /// <summary>
+        /// An AES cryptographic algorithm backed by HSM.
```

Comment:
nit: If HSM here refers to Hardware Security Module, would "backed by an HSM" or "backed by a Hardware Security Module (HSM)" be better?